plugins/become/ksusu.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    name: ksusu
+    short_description: Kerberos substitute (ksu) user followed by substitute user (su)
+    description:
+        - This become plugins allows your remote/login user to execute commands as another user by calling C(su) through the C(ksu) utility.
+    author:
+        - Ed Schaller (@schallee)
+    version_added: 6.2.0
+    options:
+        become_user_ksu:
+            description: User you use with C(ksu) to run C(su).
+            default: root
+            ini:
+              - section: privilege_escalation
+                key: become_user_ksu
```

Comment:
I still don't think we should add new configuration entries to the `privilege_escalation` section.

Being able to configure this in another section of `ansible.cfg` should be sufficient.